wremnants/theory_tools.py

```diff
@@ -151,6 +151,7 @@ def define_prefsr_vars(df):
     df = df.Define("genlanti", "ROOT::Math::PtEtaPhiMVector(GenPart_pt[prefsrLeps[1]], GenPart_eta[prefsrLeps[1]], GenPart_phi[prefsrLeps[1]], GenPart_mass[prefsrLeps[1]])")
     df = df.Define("genV", "ROOT::Math::PxPyPzEVector(genl)+ROOT::Math::PxPyPzEVector(genlanti)")
     df = df.Define("ptVgen", "genV.pt()")
+    df = df.Define("ptqVgen", "genV.pt()/genV.mass()")
```

Comment:
perhaps here and wherever else it appears, a better name for the new variable could be ptOverQVgen or ptOverMassVgen, because ptqVgen is extremely easy to confuse with the standard pt